tests/tools/plugins/ssl_hook_test.cc

```diff
@@ -295,7 +295,7 @@ setup_callbacks(TSHttpTxn txn, int preaccept_count, int client_hello_count, int
   TSDebug(PN, "Setup callbacks pa=%d client_hello=%d client_hello_imm=%d sni=%d cert=%d cert_imm=%d pa_delay=%d", preaccept_count,
           client_hello_count, client_hello_count_immediate, sni_count, cert_count, cert_count_immediate, preaccept_count_delay);
   for (i = 0; i < preaccept_count; i++) {
-    cb = TSContCreate(&CB_Pre_Accept, TSMutexCreate());
```

Comment:
Is this supposed to be part of this change?

Reply:
Yeah this is an intended change.